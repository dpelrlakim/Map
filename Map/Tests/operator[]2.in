[] a 10
[] a 20
[] a 0
[] a 15
[] a -5
[] a 5
[]= a -2147483648 -2147483648
[]= a 2147483647 2147483647
erase a 10
at a 10
erase a -2147483648
at a -2147483648
erase a 2147483647
at a 2147483647
erase a 15
at a 15
erase a 20
at a 20
erase a 5
at a 5
erase a 0
at a 0
erase a -5
at a -5
