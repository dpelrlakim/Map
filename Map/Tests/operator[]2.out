0
a: (10,0)
0
a: (10,0) (20,0)
0
a: (0,0) (10,0) (20,0)
0
a: (0,0) (10,0) (15,0) (20,0)
0
a: (-5,0) (0,0) (10,0) (15,0) (20,0)
0
a: (-5,0) (0,0) (5,0) (10,0) (15,0) (20,0)
a: (-2147483648,-2147483648) (-5,0) (0,0) (5,0) (10,0) (15,0) (20,0)
a: (-2147483648,-2147483648) (-5,0) (0,0) (5,0) (10,0) (15,0) (20,0) (2147483647,2147483647)
a: (-2147483648,-2147483648) (-5,0) (0,0) (5,0) (15,0) (20,0) (2147483647,2147483647)
Exception
a: (-2147483648,-2147483648) (-5,0) (0,0) (5,0) (15,0) (20,0) (2147483647,2147483647)
a: (-5,0) (0,0) (5,0) (15,0) (20,0) (2147483647,2147483647)
Exception
a: (-5,0) (0,0) (5,0) (15,0) (20,0) (2147483647,2147483647)
a: (-5,0) (0,0) (5,0) (15,0) (20,0)
Exception
a: (-5,0) (0,0) (5,0) (15,0) (20,0)
a: (-5,0) (0,0) (5,0) (20,0)
Exception
a: (-5,0) (0,0) (5,0) (20,0)
a: (-5,0) (0,0) (5,0)
Exception
a: (-5,0) (0,0) (5,0)
a: (-5,0) (0,0)
Exception
a: (-5,0) (0,0)
a: (-5,0)
Exception
a: (-5,0)
a:
Exception
a:
